frontend/app/module/election/page/ElectionStatusPage.tsx

```diff
@@ -67,21 +67,28 @@ function getTableHeaderForCategory(category: StatusCategory): ReactNode {
       {t("progress")}
     </Table.Column>
   );
-  // TODO: Needs to be updated when second entry is implemented
-  if (category === "unfinished") {
-    return <CategoryHeader />;
-  } else if (category === "in_progress") {
-    return <CategoryHeader>{[progressColumn]}</CategoryHeader>;
-  } else if (category === "definitive") {
-    return <CategoryHeader>{[finishedAtColumn]}</CategoryHeader>;
-  } else {
-    return <CategoryHeader></CategoryHeader>;
+
+  switch (category) {
+    case "unfinished":
+      return <CategoryHeader />;
+    case "in_progress":
+      return <CategoryHeader>{[progressColumn]}</CategoryHeader>;
+    case "first_entry_finished":
+    case "definitive":
+      return <CategoryHeader>{[finishedAtColumn]}</CategoryHeader>;
+    default:
+      return <CategoryHeader></CategoryHeader>;
   }
 }
 
 function getTableRowForCategory(category: StatusCategory, polling_station: PollingStationWithStatus): ReactNode {
   // TODO: future `errors_and_warnings` status should be added to showBadge array
-  const showBadge = ["first_entry_unfinished", "first_entry_in_progress"];
+  const showBadge: PollingStationStatus[] = [
+    "first_entry_unfinished",
+    "first_entry_in_progress",
+    "second_entry_unfinished",
```

Comment:
This badge lacks the background color for me. The color is present for "1e invoer".
![image](https://github.com/user-attachments/assets/f15f6542-9af4-436d-b4e9-a5b632187266)